microprofile/WFLY-17510-remove-microprofile-opentracing.adoc

```diff
@@ -0,0 +1,100 @@
+= WFLY-17510: Remove MicroProfile OpenTracing
+:author:            Jason Lee
+:email:             jasondlee@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+WildFly has long offered MicroProfile OpenTracing as part of its overall MicroProfile support. Since its inclusion in WildFly, however, OpenTracing has merged with OpenCensus to form https://opentelemetry.io[OpenTelemetry]. Because of this merger, the OpenTracing project itself has been abandoned, and the MicroProfile Working Group has decided to remove OpenTracing from the platform specification, replacing it with MicroProfile Telemetry, which wraps and exposes https://opentelemetry.io[OpenTelemetry]. The effort under this JIRA will cover only the effort to remove MicroProfile OpenTracing (MicroProfile OpenTracing) from the server and migrate installations to MicroProfile Telemetry.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.redhat.com/browse/WFLY-17510[WFLY-17510]
+
+=== Related Issues
+
+* https://issues.redhat.com/browse/WFLY-17156[WFLY-17156]
+* https://issues.redhat.com/browse/EAP7-2002[EAP7-2002]
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+=== Testing By
+* [x] Engineering
+* [ ] QE
+
+=== Affected Projects or Components
+
+All the code changes will be confined to the MicroProfile OpenTracing extension and subsystem, which includes the Java code, Galleon modules/layers, etc.
+
+=== Other Interested Projects
+
+OpenTelemetry/MicroProfile Telemetry are involved but unchanged.
+
+=== Relevant Installation Types
+* [x] Traditional standalone server (unzipped or provisioned by Galleon)
+* [ ] Managed domain
+* [x] OpenShift s2i
+* [x] Bootable jar
+
+== Requirements
+
+=== Hard Requirements
+
+MicroProfile OpenTracing will be migrated to an admin-only module
```

Comment:
" and removed from standard configs" maybe?

Reply:
Yes. Good catch.